static/js/components/email/lib_test.js

```diff
@@ -273,16 +273,153 @@ describe('Specific email config', () => {
         });
       });
 
-      describe('findTerms', () => {
-        it('should return filters', () => {
-          let filters = findTerms(JSON.parse(queryFilters));
-          assert.equal(filters.length, 1);
-          assert.deepEqual(filters, [{
-            "program.enrollments.payment_status": "Paid"
-          }]);
+      describe('findFilters', () => {
+        it('should return an empty list if passed an empty object', () => {
+          let filters = findFilters({});
+          assert.equal(filters.length, 0);
+        });
+
+        it('should return an empty list if passed an object without any "term" and "range" key on it', () => {
+          const query = `{
+            "bool": {
+              "must":[
+                {
+                  "program.id":1
+                }
+              ]
+            }
+          }`;
+          let filters = findFilters(JSON.parse(query));
+          assert.equal(filters.length, 0);
+        });
+
+        it('should return a list of filters defined at the top-level', () => {
+          const query = `{
+            "bool": {
+              "must":[
+                {
+                  "range": {
+                    "program.grade_average": {
+                      "gte": 0,
+                      "lte": 81
+                    }
+                  }
+                },
+                {
+                  "term": {
+                    "profile.birth_country": "US"
+                  }
+                },
+                {
+                  "term": {
+                    "profile.country": "US"
+                  }
+                }
+              ]
+            }
+          }`;
+          let filters = findFilters(JSON.parse(query));
+          assert.equal(filters.length, 3);
```

Comment:
instead of just checking the length here, could you use `assert.deepEqual` to make sure the results are exactly what we expect?